index.ts

```diff
@@ -1,2 +1,46 @@
-import CashBalances from './data';
+import CashBalances, {CashBalanceReport, ReportSection, Account} from './data';
+
+interface ReportAccountTotals {
+  highest_account_name: string
+  lowest_account: string
+  average: any
```

Comment:
```suggestion
  average: number
```
We should be using a lint rule set like `tslint:recommended` that would catch this.